src/ui/endpoint_selection_plugin.h

```diff
@@ -31,6 +31,7 @@ class EndPoint_Selection_Menu : public FishUIViewerPlugin {
 
     Eigen::Vector4f old_viewport;
 
+    float scaling_factor = 2.f;
```

Comment:
Are we using this scaling factor at all? If not, let's delete it. If so, let's wrap it in an `ifdef`, rename it and add a comment.